modules/tensor_mechanics/test/tests/inclined_bc/tests

```diff
@@ -4,15 +4,24 @@
     input = 'inclined_bc_2d.i'
     exodiff = 'inclined_bc_2d_out.e'
     issues = '#13128'
-    design = 'PenaltyInclinedBC.md'
-    requirement = 'The TensorMechanics module shall have the capabilty to enforce inclined boundary conditions on a 2D model using penalty method.'
+    design = 'PenaltyInclinedNoDisplacementBC.md'
+    requirement = 'The TensorMechanics module shall have the capabilty to enforce inclined boundary conditions on a 2D model using a penalty method.'
   [../]
   [./3D]
     type = Exodiff
     input = 'inclined_bc_3d.i'
     exodiff = 'inclined_bc_3d_out.e'
     issues = '#13128'
-    design = 'PenaltyInclinedBC.md'
-    requirement = 'The TensorMechanics module shall have the capabilty to enforce inclined boundary conditions on a 3D model using penalty method.'
+    design = 'PenaltyInclinedNoDisplacementBC.md'
+    requirement = 'The TensorMechanics module shall have the capabilty to enforce inclined boundary conditions on a 3D model using a penalty method.'
+  [../]
+  [./3D_action]
+    type = Exodiff
+    input = 'inclined_bc_action.i'
+    exodiff = 'inclined_bc_3d_out.e'
+    prereq = '3D'
+    issues = '#13128'
+    design = 'InclinedNoDisplacementBCAction.md'
+    requirement = 'The InclinedNoDisplacement boundary condition action shall create the objects needed to apply inclined boundary conditions on a 3D model using a penalty method.'
```

Comment:
This test requires superLU - make sure you add superlu = true in this test spec.